nix/plutus/latex-docs.nix

```diff
@@ -0,0 +1,146 @@
+{ nix, inputs, pkgs, l, ... }:
+
+let
+
+  latex-docs = l.flip l.mapAttrValues nix.plutus.build-latex-doc {
+
+    cost-model-notes = {
```

Comment:
fine I guess. I kind of did like the std thing of one package per file, but whatever